### ISSUE TITLE
pastehub-clientSync fails with RuntimeError Exception for cloesd GDBM

### ISSUE THREAD

**Issue by kiyoka** [kiyoka/pastehub#2]
```
Info: masterList lines = 100  7199Bytes
Info: download 1 records.
Info:   upload 0 records.
Info:   send signal to Emacs.
Emacs: no process found
Info: server is stable:    1341707393=2012-07-08.00:29:52
/usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/lib/pastehub/db.rb:41:in `[]': closed GDBM file (RuntimeError)
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/lib/pastehub/db.rb:41:in `getValue'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/lib/pastehub/client.rb:128:in `serverHasNew?'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/lib/pastehub/client.rb:93:in `block (3 levels) in wait_notify'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/protocol.rb:395:in `call_block'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/protocol.rb:386:in `<<'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/protocol.rb:94:in `read'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2776:in `read_chunked'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2750:in `read_body_0'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2710:in `read_body'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/lib/pastehub/client.rb:91:in `block (2 levels) in wait_notify'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1322:in `block (2 levels) in transport_request'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2671:in `reading_body'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1321:in `block in transport_request'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1316:in `catch'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1316:in `transport_request'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1293:in `request'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/lib/pastehub/client.rb:89:in `block in wait_notify'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:745:in `start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:557:in `start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/lib/pastehub/client.rb:87:in `wait_notify'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/bin/pastehub-clientSync:65:in `syncMain'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/bin/pastehub-clientSync:119:in `block in main'
```

**Comment by kiyoka** [comment]
I face this error, today.
The error message is ...
I'v done copy-operation in Emacs-24, then pastehub-clientSync was failed .

```
Info: server is stable:    1342880094=2012-07-21.14:14:54
waiting...
Info: server is stable:    1342880094=2012-07-21.14:14:54
/usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/lib/pastehub/localdb.rb:41:in `[]': closed GDBM file (RuntimeError)
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/lib/pastehub/localdb.rb:41:in `getValue'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/lib/pastehub/client.rb:128:in `serverHasNew?'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/lib/pastehub/client.rb:93:in `block (3 levels) in wait_notify'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/protocol.rb:395:in `call_block'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/protocol.rb:386:in `<<'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/protocol.rb:94:in `read'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2776:in `read_chunked'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2750:in `read_body_0'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2710:in `read_body'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/lib/pastehub/client.rb:91:in `block (2 levels) in wait_notify'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1322:in `block (2 levels) in transport_request'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:2671:in `reading_body'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1321:in `block in transport_request'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1316:in `catch'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1316:in `transport_request'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:1293:in `request'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/lib/pastehub/client.rb:89:in `block in wait_notify'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:745:in `start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:557:in `start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/lib/pastehub/client.rb:87:in `wait_notify'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/bin/pastehub-clientSync:69:in `syncMain'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.2/bin/pastehub-clientSync:131:in `block in main'
```

**Comment by kiyoka** [comment]
I suspect that it conflicts for writing the gdbm file.
pastehub-clientPost and pastehub-clientSync will try to open for write opration.

**Comment by kiyoka** [comment]
This issue was already resolved on 0.1.3 release
